README.md

```diff
@@ -1,15 +1,23 @@
-# AnalyzeJournal Activity #
+AnalyzeJournal Activity
+===============
+This Activity charts data from the Sugar Journal and gives you a visual representation of Disk usage, Activity usage and Turtle data usage using a vertical bar chart, a horizontal bar chart or a pie chart.
 
-This Activity charts data from the Sugar Journal:
-
-1. free space
-2. activity usage
-3. turtle block usage
-
-This activity gives you the possibility to graphically, the journal usage.
-
-### To Know more about AnalyzeJournal and Sugar, Please refer to ###
```

Comment:
Is it necessary to remove this?

Reply:
It wasn't exactly removed just rephrased.

Reply:
Ig most of the information is not clearly mentioned in the new Readme. You can mark it as resolved if you feel its right.

Reply:
If you have any suggestions or a better description, I would love to add it if its well-fitting! Otherwise I feel the readme is alright! :)